src/Evolution/Systems/Ccz4/Christoffel.hpp

```diff
@@ -0,0 +1,37 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace Ccz4 {
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the conformal spatial christoffel symbols of the second kind.
+ *
+ * \details Computes the christoffel symbols as:
+ * \f{align}
+ *     \bar{\Gamma}^k_{ij} &= \bar{\gamma}^{ij} (D_{ijl} + D_{jil} - D_{lij})
```

Comment:
I think `\bar` -> `\tilde` no?

LHS `\gamma^{ij}` should be `\gamma^{kl}`

Reply:
The reason I used `\bar` instead was because of @nilsleiffischer 's review request in a previous PR [here](https://github.com/sxs-collaboration/spectre/pull/3502#discussion_r705065635), where it's requested because it seems to be consistent with the Baumgarte & Shapiro book. I'm happy to do either notation, I just continued with using `\bar` to be consistent with that review request. If you feel differently, maybe you and @nilsleiffischer can discuss and let me know which you prefer :+1:

Reply:
I think anything in CCZ4 should be consistent with the paper, otherwise it'll almost certainly lead to confusion (as you saw by me being confused :) )